packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -291,7 +322,17 @@ abstract class WidgetsBinding extends BindingBase implements GestureBinding, Ren
       debugShortDescription: '[root]',
       child: app
     ).attachToRenderTree(buildOwner, renderViewElement);
+    assert(() {
+      if (debugPrintBeginFrameBanner)
+        debugPrint('━━━━━━━┫ Begin Preemptive Frame ┣━━━━━━━');
```

Comment:
not sure what the right word is. I mean a frame that we're doing before the vsync because we want to have extra time to do it so that when the vsync actually happens, we're ready and got a few extra milliseconds of work done already.